src/ConstructiveSolidGeometry/ConstructiveSolidGeometry.jl

```diff
@@ -11,6 +11,7 @@ module ConstructiveSolidGeometry
     using RecipesBase
     using Rotations
     using StaticArrays
+    using Unitful
```

Comment:
I am not sure if we want this here. Sure, In general, general unit support would be really nice inside the CSG but it might makes everything more complicated. 

I have to think about this.